lib/Extractor/Candidates.cpp

```diff
@@ -151,7 +156,11 @@ Inst *ExprBuilder::makeArrayRead(Value *V) {
   if (Opts.NamedArrays)
     Name = V->getName();
   unsigned Width = DL.getTypeSizeInBits(V->getType());
-  return IC.createVar(Width, Name);
+  // Compute known bits
+  APInt KnownZero(Width, 0), KnownOne(Width, 0);
```

Comment:
My bad ! 2-argumnts constructor is never safe to be used as mentions the APInt.h and 3-argument constructor is the public constructor to be used always. Sorry about leaving it like that earlier.